lib/world/sequential.rb

```diff
@@ -72,6 +72,13 @@ def initialize(datafile = nil)
 
         @city_populations = JSON.parse(datafile) if datafile
 
+        # import hospitals
+        file = File.read './config/healthcare_facilities.json'
+        providers = JSON.parse(file)
+        providers.each do |_provider_name, provider_stats|
```

Comment:
Oh, I see... you are caching it in the constructor.

Why not just have the class load the definitions? Then you can do it in one place and have it self-contained like the modules.

It seems like you this... maybe so you can override the list in the unit tests. Blah.